cmd/dep/init_test.go

```diff
@@ -144,3 +147,125 @@ const Qux = "yo yo!"
 	// vendor should have been created & populated
 	h.MustExist(h.Path("src/" + root + "/vendor/github.com/Sirupsen/logrus"))
 }
+
+// TestCaseInsentitive is test for Windows. This should work even though set
+// difference letter cases in GOPATH.
+func TestCaseInsentitive(t *testing.T) {
```

Comment:
I think I'd still prefer to get away with a narrower test - more unit, less integration. 

With the way dep works right now, there's really only the one place that we use GOPATH at all (in `Ctx.LoadProject()`, though that's after setting it in `NewContext()`), and that's unlikely to change. 

Maybe it's worth having this integration-level check as well (and I'll think about that some more), but for now, I think that ensuring the specific paths around `*Ctx` are working wrt case insensitivity is what we really need.